extra_data/components.py

```diff
@@ -1159,7 +1157,7 @@ def __init__(self, data: DataCollection, detector_name=None, modules=None,
             self.n_modules = int(n_modules)
         else:
             # For JUNGFRAU modules are indexed from 1
-            self.n_modules = self.max_module
+            self.n_modules = max(self.source_to_modno.values())
```

Comment:
But

Reply:
Sorry, now I see you have fixed it in the later commit https://github.com/European-XFEL/EXtra-data/pull/62/commits/e2565ff8cfc87d9aed758a5f1e5e0e28027e88ad =)